src/frontend/src/flows/recovery/displaySeedPhrase.json

```diff
@@ -1,13 +1,14 @@
 {
   "en": {
-    "title": "Store Recovery Phrase",
-    "header": " If you lose access to your devices, use your recovery phrase to access your Internet Identity.",
-    "your_recovery_phrase": "Your recovery phrase",
+    "title": "Safely Store your Recovery Phrase",
```

Comment:
shouldn't it be "store" (lower letter as in the verb)?

Reply:
so "Your" not "your" then according to https://capitalizemytitle.com/ 😉

Reply:
I'll forward this to the relevant people 😄 I don't care one bit